components/tools/OmeroWeb/omeroweb/webclient/show.py

```diff
@@ -896,6 +846,87 @@ def paths_to_object(conn, experimenter_id=None, project_id=None,
     return paths
 
 
+def paths_to_well_image(conn, params, well_id=None, image_id=None,
+                        acquisition_id=None,
+                        plate_id=None, screen_id=None, experimenter_id=None,
+                        orphanedImage=False):
+
+    qs = conn.getQueryService()
+    service_opts = deepcopy(conn.SERVICE_OPTS)
+    q = '''
+        select coalesce(sowner.id, plowner.id, aowner.id, wsowner.id),
+               slink.parent.id,
+               plate.id,
+               acquisition.id,
+               well.id
+        from WellSample wellsample
+        left outer join wellsample.details.owner wsowner
+        left outer join wellsample.plateAcquisition acquisition
+        left outer join wellsample.details.owner aowner
```

Comment:
The same object `wellsample.details.owner` is being applied to two separate aliases (`wsowner` and `aowner`) here. As `details` are always present and the base of your `JOIN`s is `wellsample` you also don't need `left outer` here.

Maybe add a failing test case for this condition and then fix the bug? Or is this just completely irrelevant and can be removed from the query?